extensions/wikia/ImageServing/drivers/ImageServingDriverMainNS.class.php

```diff
@@ -56,16 +56,20 @@ private function buildMimeTypesBlacklist() {
 		}
 	}
 
-	protected function loadImagesFromDb( $articleIds = array() ) {
+	protected function loadImagesFromDb( $articleIds = [ ] ) {
 		wfProfileIn( __METHOD__ );
 
-		//load infobox images at start
-		$this->loadImagesFromInfoboxes( $articleIds );
+		//load infobox images at start, set number of images from infoboxes
+		$order = $this->loadImagesFromInfoboxes( $articleIds );
```

Comment:
I know it's used as order in addImage, though I think it'll be much clearer if it's named as above comment states, along the lines of numberOfImages